numba/tests/test_parfors.py

```diff
@@ -3981,6 +3983,107 @@ def test_issue_due_to_max_label(self):
         self.assertIn("TEST PASSED", out.decode())
 
 
+class TestParforChunksizing(TestCase):
+    """
+    Tests miscellaneous parts of ParallelAccelerator use.
+    """
+    _numba_parallel_test_ = False
+
+    @skip_parfors_unsupported
+    def test_python_parallel_chunksize_basic(self):
```

Comment:
Should all these tests be context managed or include a `setUp/tearDown` which sets the chunksize to 0? This is so that if e.g. a test fails after it has set the chunksize to some non-default, that subsequent tests are not impacted by this state.

Reply:
@stuartarchibald I'm not sure what to do here.  The context manager that would do this is what we are testing in many cases.  Could you make this change as you'd like it to be?  Thanks.